opendebates/models.py

```diff
@@ -28,6 +28,19 @@ class Meta:
         verbose_name_plural = _("categories")
 
 
+class SiteMode(CachingMixin, models.Model):
+    show_question_votes = models.BooleanField(default=True, blank=True)
+    show_total_votes = models.BooleanField(default=True, blank=True)
+    allow_sorting_by_votes = models.BooleanField(default=True, blank=True)
+    allow_voting_and_submitting_questions = models.BooleanField(default=True, blank=True)
+    debate_time = models.DateTimeField(
+        default=datetime.datetime(2099, 1, 1),
+        help_text="Enter time that debate starts in timezone %s" % settings.TIME_ZONE,
```

Comment:
Having Model `help_text` vary based on a setting means that Django will require a migration anytime the setting is changed. I don't think that's an issue here, but just wanted to mention it.